app/_src/policies/virtual-outbound.md

```diff
@@ -41,6 +41,33 @@ inbound:
 
 and a virtual outbound with this definition:
 
+{% tabs default-test-virtualoutbound useUrlFragment=false %}
+{% tab default-test-virtualoutbound Kubernetes %}
```

Comment:
we have this `policy_yaml` function I think we should use: https://github.com/kumahq/kuma-website/blob/0b726a69960f8f5041a884902df58ea24a3604b0/jekyll-kuma-plugins/lib/jekyll/kuma-plugins/liquid/tags/policyyaml.rb#L1 - it takes `universal` format

Reply:
it doesn't work on the legacy policies